config/channel/consolidated/configmaps/leader-election.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: config-leader-election-kafka
+  name: config-leader-election
```

Comment:
This will override core's version. Are we sure we want to include a copy in this repo?